libhesai/Lidar/lidar_types.h

```diff
@@ -133,6 +133,8 @@ struct LidarDecodedPacket
     uint16_t spin_speed;
     uint8_t lidar_state;
     uint8_t work_mode;
+    uint8_t column_resolution;
```

Comment:
`uint8_t` is enough for resolution?

Reply:
yes, resolution is about 160x120.
so row is 120 , and column is around 160.
i dont know why but this node publish more than 160th column, though...